Add compression instruction for the Datadog Agent (#6088)
DataDog/documentation: content/en/agent/logs/_index.md

To send logs over HTTPS with the Datadog Agent 6.14+, add the following in the Agent's [main configuration file][4] (`datadog.yaml`):

{{< tabs >}}
{{% tab "Compression enabled" %}}

```
logs_config:
  use_http: true
  use_compression: true
  compression_level: 6
```

Or set the `DD_LOGS_CONFIG_USE_HTTP` and `DD_LOGS_CONFIG_USE_COMPRESSION` environment variables to `true`.
The `compression_level` parameter (or `DD_LOGS_CONFIG_COMPRESSION_LEVEL` environment variable) accept value from 0 (no compression) to 9 (maximum compression but higher resource usage). The default value is 6.

> Then restart the Agent to sends logs through HTTPS to `agent-http-intake.logs.datadoghq.com` (US site) or `agent-http-intake.logs.datadoghq.eu` (EU site) on port 443.

Reviewer: is the host name relevant here?  shouldn't setting the  "site" be sufficient?

Author: Indeed setting site is sufficient to choose to send data to US or EU site.
The goal of highlighting the domain the agent send the logs to is for users that whitelist this in their firewall policy.

To send logs over HTTPS with the Datadog Agent 6.14+, add the following in the Agent's [main configuration file][4] (`datadog.yaml`):

{{< tabs >}}
{{% tab "Compression enabled" %}}

```
logs_config:
  use_http: true
  use_compression: true
  compression_level: 6
```

Or set the `DD_LOGS_CONFIG_USE_HTTP` and `DD_LOGS_CONFIG_USE_COMPRESSION` environment variables to `true`.

> The `compression_level` parameter (or `DD_LOGS_CONFIG_COMPRESSION_LEVEL` environment variable) accept value from 0 (no compression) to 9 (maximum compression but higher resource usage). The default value is 6.

Reviewer 2: ```suggestion
The `compression_level` parameter (or `DD_LOGS_CONFIG_COMPRESSION_LEVEL` environment variable) accepts values from 0 (no compression) to 9 (maximum compression but higher resource usage). The default value is 6.
```

> See the [Datadog Agent overhead section][10] for more information about the agent usage when compression is enabled.

Reviewer 2: ```suggestion
See the [Datadog Agent overhead section][10] for more information about Agent resource usage when compression is enabled.
```

> Then restart the Agent to sends logs through HTTPS to `agent-http-intake.logs.datadoghq.com` (US site) or `agent-http-intake.logs.datadoghq.eu` (EU site) on port 443.

Reviewer 3: ```suggestion
Then restart the Agent to sends logs through HTTPS to `agent-http-intake.logs.datadoghq.com` (US site) or `agent-http-intake.logs.datadoghq.eu` (EU site) on port `443`.
```

[8]: /developers/metrics/custom_metrics
[9]: /tagging
[10]: /agent/proxy

> [11]: https://docs.datadoghq.com/agent/basic_agent_usage/?tab=agentv6#agent-overhead

Reviewer 3: ```suggestion
[11]: /agent/basic_agent_usage/?tab=agentv6#agent-overhead
```
